java-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/contexts/FieldAccessContext.java

```diff
@@ -91,4 +94,15 @@ public FieldAccessContext(FieldAccessExpr wrappedNode, TypeSolver typeSolver) {
             return getParent().solveSymbolAsValue(name, typeSolver);
         }
     }
+
+    public SymbolReference<ResolvedFieldDeclaration> solveField(String name, TypeSolver typeSolver) {
+        Collection<ResolvedReferenceTypeDeclaration> rrtds = findTypeDeclarations(Optional.of(wrappedNode.getScope()), typeSolver);
+        for (ResolvedReferenceTypeDeclaration rrtd : rrtds) {
+            try {
+                return SymbolReference.solved(rrtd.getField(wrappedNode.getName().getId()));
+            } catch (Throwable t) {
+            }
```

Comment:
We could add a comment explaining why we want to silent errors here and maybe limit to specific exceptions